### DISCUSSION TITLE
Adjustments to the PDF Docs

### DISCUSSION THREAD

**Discussion by dodiego** [apache/age#36] [General] [answered]
Hey folks,

After reading through the PDF docs, I noticed errors:

- Page 19:

![image](https://user-images.githubusercontent.com/2087311/110253834-22fa7680-7f6b-11eb-9a4a-cf6171ac19b4.png)

- Page 23:

![image](https://user-images.githubusercontent.com/2087311/110254007-f98e1a80-7f6b-11eb-8deb-e633b38d0a24.png)

- Page 31

![image](https://user-images.githubusercontent.com/2087311/110254065-3fe37980-7f6c-11eb-984f-1367a0883895.png)

- Across the document, here are my suggestions:
  - There are mixed terminologies, sometimes the document mentions "nodes" and "relationships" and other times it says "vertexes" and "edges". We should use only one pair or the other, not both.
  - The document uses "vertex labels" and "edge types" terminologies, it is best to only use the word "types" or only the word "labels", not both, since they mean the same thing, just associated with different entities (one labels vertexes, the other labels edges)

**Comment by pdpotter** [comment, marked as answer]
Thank you for your remarks!

The documentation has been converted to a website (https://age.apache.org/docs/master/index.html) generated from Markdown, making it easier to indicate specific issues and create pull requests. I fixed the issues you reported. New issues can be reported at https://github.com/apache/incubator-age-website/issues
